README.md

```diff
@@ -1105,6 +1108,89 @@ However, the following rules of thumb are useful to keep in mind:
   Alternatively, the entire OrientDB class and all edges that may point to it may be omitted
   entirely from the GraphQL schema.
 
+## Query Cost Estimation
+
+Some GraphQL queries executed over a database may use a lot of memory and/or have a long execution
```

Comment:
nit: The problem is not that the queries use a lot of memory. The problem is that they use too much memory, (by some definition of "too much". Therefore, consider rewriting this to "Some GraphQL queries executed over a database use too much memory and/or take too long to run."